src/main/resources/emails/en/collaborationInvite.html

```diff
@@ -169,109 +169,6 @@ <h1 style="Margin:0;Margin-bottom:10px;color:#005589;font-family:Helvetica,Arial
-                                                                  <center data-parsed=""
-
-                                                                  style="min-width:0;width:100%"><a
-
-                                                                  href="http://www.flukebook.org"
-
-                                                                  align="center"
-
-                                                                  class="float-center"
-
-                                                                  style="Margin:0;border:0 solid #3adb76;border-radius:3px;color:#fefefe;display:inline-block;font-family:Helvetica,Arial,sans-serif;font-size:16px;font-weight:700;line-height:1.3;margin:0;padding:8px 16px 8px 16px;padding-left:0;padding-right:0;text-align:center;text-decoration:none;width:100%">Adopt
-                                                                  a whale</a></center>
-                                                                  </td>
-                                                                  </tr>
-                                                                  </tbody>
-                                                                  </table>
-                                                                  </td>
-                                                                  <td class="expander"
-
-                                                                  style="-moz-hyphens:auto;-webkit-hyphens:auto;Margin:0;border-collapse:collapse!important;color:#0a0a0a;font-family:Helvetica,Arial,sans-serif;font-size:16px;font-weight:400;hyphens:auto;line-height:1.3;margin:0;padding:0!important;text-align:left;vertical-align:top;visibility:hidden;width:0;word-wrap:break-word"><br />
-                                                                  </td>
-                                                                  </tr>
-                                                                </tbody>
-                                                              </table>
-                                                            </th>
-                                                          </tr>
-                                                        </tbody>
-                                                      </table>
-                                                    </th>
-                                                  </tr>
-                                                </tbody>
-                                              </table>
-                                            </td>
-                                          </tr>
-                                        </tbody>
-                                      </table>
-                                    </td>
-                                  </tr>
-                                </tbody>
-                              </table>
```

Comment:
just going to trust that the tags are all balanced after this removal.  :sweat_smile: